### PR TITLE
1404 fix focus element

### PR DESCRIPTION
**Pull request by joehall-tw**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [X] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

### Bug
The user would be anchored to the footer when clicking the next page of results (cards) in the dashboard. 

To reproduce:
- Go to the Ask-VA form.
- Select and fill out values to progress through steps
- Begin to click "back" or "continue" quickly and/or randomly
- The user will be pulled to the bottom of the footer rather than to the top of the form.

### Fix

currently a call to `focusElement` was used to focus the user back to the top of the form. The element being focused was the ProgressBar below the form header. Sometimes the header element would not be rendered by time the focusElement was called, causing the focus to push the user down to the footer.

The `focusElement` call was replaced with `scrollTo`. Using `scrollTo` during testing consistently returned the view to the top of the form.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#0000
- _Link to previous change of the code/bug (if applicable)_
This PR is based on the bug described in department-of-veterans-affairs/ask-va#1404
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done

- _Describe what the old behavior was prior to the change_

Moving from page to page, the scroll wouldn't reset to the top of page.

- _Describe the steps required to verify your changes are working as expected_

Click to the next page of results (cards) in the dashboard. When clicking to the next page, the user should arrive at the top of the page per usual.

- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

Manual testing. Successfully saw scroll to top of page.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

N/A

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [X] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [X] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [X] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

No concerns. Replaced one platform utility with another.

### CODE REVIEW

**Review by ediiotero — DISMISSED** [review]
Looks good

**Comment by KaemonIsland** [comment]
Added more notes discussing how to reproduce the bug, and how the applied fix works.

File: src/applications/ask-va/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Temp Change
+
+This is a simple change so that I can create a new branch for a new dev on our team.
```
Comment (CBonade) [inline src/applications/ask-va/README.md:3]: If this pull request is final, should we remove this temporary change?

**Review by CBonade — DISMISSED** [review]
Just one concern with a temporary change. Will re-review once addressed.

File: src/applications/ask-va/README.md (hunk 1 above)
Comment (KaemonIsland) [reply src/applications/ask-va/README.md]: Right, this has been removed.

**Review by ediiotero — APPROVED** [review]
LGTM

**Review by ataker — APPROVED** [review]

**Pull request merged**